php/class-media.php

```diff
@@ -1935,7 +1935,7 @@ public function get_breakpoint_options( $attachment_id ) {
 		$settings    = $this->settings->get_setting( self::MEDIA_SETTINGS_SLUG )->get_value();
 
 		if ( 'on' === $settings['enable_breakpoints'] && wp_attachment_is_image( $attachment_id ) ) {
-			$meta = wp_get_attachment_metadata( $attachment_id );
+			$meta = wp_get_attachment_metadata( $attachment_id, true );
```

Comment:
@DavidCramer 

We do still have other instances where we use the filtered `wp_get_attachment_metadata`. Is there as opportunity to normalize this, or those cases are actually supposed to factor in any filters?